source/utilities.tex

```diff
@@ -14022,10 +14022,10 @@
 \begin{itemize}
 \item \tcode{FD} is the type \tcode{decay_t<F>},
 \item \tcode{fd} is an lvalue of type \tcode{FD} constructed from \tcode{std::forward<F>(f)},
-\item \tcode{Ti} is the $i^{th}$ type in the template parameter pack \tcode{BoundArgs},
-\item \tcode{TiD} is the type \tcode{decay_t<Ti>},
+\item $\tcode{T}_i$ is the $i^{th}$ type in the template parameter pack \tcode{BoundArgs},
+\item $\tcode{TD}_i$ is the type \tcode{decay_t<$\tcode{T}_i$>},
 \item \tcode{ti} is the $i^{th}$ argument in the function parameter pack \tcode{bound_args},
```

Comment:
I think here we would also like `\tcode{t}_i` for the arguments.

I'm beginning to think that this changeset is getting too large. I had already been considering each of these passages individually, and each of them requires more care and finetuning that this blanket search-and-replace approach, I think. My recommendation would be to separate the strictly non-observable changes (`\tcode{T}$_i$` to `$\tcode{T}_i$`) from the rest, and revise each section separately for the more invasive changes. For example, #1251 is such a revision of one particular section.